### PR TITLE
Fast remove small regions

### PR DESCRIPTION
**Pull request by e-koch**
My previous implementation was terribly slow. skimage has a wonderfully fast version though, so it makes sense to just switch to that.

```radio_beam``` will [soon include a tophat kernel](https://github.com/radio-astro-tools/radio_beam/pull/22) that we can base the pixel area of the beam off of.

### CODE REVIEW

**Comment by e-koch** [comment]
@keflavich - Ready to be reviewed.

#38 has become more complicated than first anticipated, so I'd like to merge this in soon.

File: signal_id/mask.py [hunk 1]
```diff
@@ -413,19 +413,15 @@ def remove_small_regions(self, area_threshold=None, beam=None,
             # Eventually the unit should be included in get_pixel_scales
             # but this requires additional workarounds when used for Beam
             # objects
-            pixscale = get_pixel_scales(self._wcs) * u.deg
+            pixscale = get_pixel_scales(self._wcs)
+
             # Now get the pixel beam area
-            area_threshold = (beam.sr.to(u.deg**2) / pixscale**2.).value
+            area_threshold = (beam.as_tophat_kernel(pixscale).array > 0).sum()
```
Comment (keflavich) [inline signal_id/mask.py]: `np.count_nonzero` is slightly more efficient than `.sum()` for boolean array summation like this.  This hint courtesy Julian Taylor

File: signal_id/utils.py [hunk 2]
```diff
@@ -138,3 +140,119 @@ def get_pixel_scales(mywcs):
     #scale = np.array([cdelt[0] * (pc[0,0]**2 + pc[1,0]**2)**0.5,
     # cdelt[1] * (pc[0,1]**2 + pc[1,1]**2)**0.5])
     #return abs(scale[0])
+
+
+def remove_small_objects(ar, min_size=64, connectivity=1, in_place=False):
+    """
+
+    Copyright (C) 2011, the scikit-image team
```
Comment (keflavich) [inline signal_id/utils.py]: Can this notice perhaps go at the bottom of the docstring?

**Comment by keflavich** [comment]
Looks fine, as far as I can tell.

**Comment by e-koch** [comment]
Addressed minor comments. Thanks!

**Pull request merged**